docs/modules/clients/pages/java-thin-client.adoc

```diff
@@ -0,0 +1,1118 @@
+= {java-standard-client-name}
```

Comment:
It's a "standalone" client, not standard

Reply:
I've created new variables and I'm using these, will highlight when new PR is up